views/random-beer.hbs

```diff
@@ -0,0 +1,20 @@
+<div>
+
+    {{#each randomBeer}}
+        {{!-- <h1>{{name}}</h1>
+      <h3>{{tagline}}</h3>  
+         {{description}} --}}
+          {{> beerpartial this}}
+           <h3>Food Pairing</h3>
+        <ul>
+         {{#each this.food_pairing}}
+         <li>{{this}}</li>
+         {{/each}}
+         </ul>
+          <h3>Brewer Tips</h3>
+         {{brewers_tips}}
+
+       {{!-- <img src={{image_url}} alt="imgPic"> --}}
+    {{/each}}
```

Comment:
An alternative solution to adding #each statement here, would be passing `beersFromApi[0]` to the view instead of `beersFromApi`. This is perfectly fine of course. 

But if you do it, you will have to replace `#each` with `#with`.